twindle-web/style.css

```diff
@@ -30,6 +30,7 @@ header a {
     align-items: center;
 }
 
+
 header nav {
     flex: 1;
```

Comment:
Any changes in this file?

Reply:
@johnjacobkenny 
No changes in any other files apart from `Thread.hbs`, The other changes were captured unexpectedly